.github/workflows/bc.yml

```diff
@@ -1,34 +1,34 @@
 on:
-  pull_request:
-    paths-ignore:
-      - 'docs/**'
-      - 'README.md'
-      - 'CHANGELOG.md'
-      - '.gitignore'
-      - '.gitattributes'
-      - 'infection.json.dist'
-      - 'phpunit.xml.dist'
-      - 'psalm.xml'
-  push:
-    branches: ['master']
-    paths-ignore:
-      - 'docs/**'
-      - 'README.md'
-      - 'CHANGELOG.md'
-      - '.gitignore'
-      - '.gitattributes'
-      - 'infection.json.dist'
-      - 'phpunit.xml.dist'
-      - 'psalm.xml'
+    pull_request:
```

Comment:
Usually yaml uses 2 spaces. I'd keep it like that.

Reply:
I use the standard ide settings, for php I used php-fig code style, but style ci always puzzles me with edits, although I adjusted the ide for style ci 😅, I'll have to look at the settings for yaml